### PR TITLE
Add a simple topic consumer tool

### PR DESCRIPTION
**Pull request by wvanbergen**
It will consume all partitions for a given topic and write the messages to STDOUT.

@Shopify/kafka

### CODE REVIEW

File: tools/kafka-console-topicconsumer/kafka-console-topicconsumer.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+	for _, partition := range partitions {
+		pc, err := c.ConsumePartition(*topic, partition, initialOffset)
+		if err != nil {
+			printErrorAndExit(69, "Failed to start consumer for partition %d: %s", partition, err)
+		}
+
+		wg.Add(1)
+		go func(partition int32, pc sarama.PartitionConsumer) {
+			defer wg.Done()
```
Comment (eapache) [inline tools/kafka-console-topicconsumer/kafka-console-topicconsumer.go]: this doesn't have to be part of the wait group, it must terminate before the messages goroutine does (except in the case of OffsetOutOfRange when I don't think it matters)

File: tools/kafka-console-topicconsumer/kafka-console-topicconsumer.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		printErrorAndExit(69, "Failed to start consumer: %s", err)
+	}
+
+	partitions, err := c.Partitions(*topic)
+	if err != nil {
+		printErrorAndExit(69, "Failed to get the list of partitions: %s", err)
+	}
+
+	var (
+		messages = make(chan *sarama.ConsumerMessage, 1)
```
Comment (eapache) [inline tools/kafka-console-topicconsumer/kafka-console-topicconsumer.go]: I'd give this a bigger buffer to reduce context-switching; sarama's default buffer size for user-facing channels is 256

**Comment by eapache** [comment]
Little things, but LGTM

**Comment by bouk** [comment]
Maybe "kafka-console-consumer" is a better name, not like there's other things to consume in Kafka 😃

**Comment by eapache** [comment]
We actually already have a `kafka-console-partitionconsumer` which only consumes a single partition, but I wouldn't mind just merging them into one `kafka-console-consumer` :)

**Comment by wvanbergen** [comment]
I think `kafka-console-consumer` should be built on top the high-level consumer once we have it.

**Comment by wvanbergen** [comment]
@Shopify/kafka Feel free to still comment on this further; there's definitely ways to improve this tool, but I had an urgent need for it, so I merge this PR.

**Pull request merged**